Makefile

```diff
@@ -72,17 +72,20 @@ docs: ## generate Sphinx HTML documentation, including API docs
 CONTENT_URL := http://research.cs.aalto.fi/pml/software/elfi/docs/0.5/
 
 notebook-docs: ## Conver notebooks to rst docs. Assumes you have them in `notebooks` directory.
-	jupyter nbconvert --to rst notebooks/quickstart.ipynb --output-dir docs
-	sed -i 's|\(quickstart_files/quickstart.*\.\)|'${CONTENT_URL}'\1|g' docs/quickstart.rst
+	jupyter nbconvert --to rst ../notebooks/quickstart.ipynb --output-dir docs
+	sed -i '' 's|\(quickstart_files/quickstart.*\.\)|'${CONTENT_URL}'\1|g' docs/quickstart.rst
```

Comment:
It's the suffix, which is a mandatory parameter on some versions of sed (esp. on Macs).